Log message:
fix the partially initialized hw_regs_t for all users

Modified files:
    linux/drivers/ide:
	ide.c 
    linux/drivers/ide/legacy:
	gayle.c q40ide.c 


Index: linux/drivers/ide/ide.c
Stats: 1 insertion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/ide/ide%2ec.diff?r1=1.1.1.79&r2=1.46
========================================================================
--- linux/drivers/ide/ide.c	9 Apr 2006 16:52:07 -0000	1.1.1.79
+++ linux/drivers/ide/ide.c	11 May 2006 23:52:06 -0000	1.46
@@ -726,6 +726,7 @@
 {
 	int i;
 
+	memset(hw, 0, sizeof(hw_regs_t));
 	for (i = 0; i < IDE_NR_PORTS; i++) {
 		if (offsets[i] == -1) {
 			switch(i) {
Index: linux/drivers/ide/legacy/gayle.c
Stats: 1 deletion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/ide/legacy/gayle%2ec.diff?r1=1.10&r2=1.11
========================================================================
--- linux/drivers/ide/legacy/gayle.c	1 Sep 2005 22:29:58 -0000	1.10
+++ linux/drivers/ide/legacy/gayle.c	11 May 2006 23:52:07 -0000	1.11
@@ -161,7 +161,6 @@
 	base = (unsigned long)ZTWO_VADDR(phys_base);
 	ctrlport = GAYLE_HAS_CONTROL_REG ? (base + GAYLE_CONTROL) : 0;
 
-	memset(&hw, 0, sizeof(hw));
 	ide_setup_ports(&hw, base, gayle_offsets,
 			ctrlport, irqport, ack_intr,
 //			&gayle_iops,
Index: linux/drivers/ide/legacy/q40ide.c
Stats: 1 insertion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/ide/legacy/q40ide%2ec.diff?r1=1.7&r2=1.8
========================================================================
--- linux/drivers/ide/legacy/q40ide.c	8 Apr 2003 19:37:55 -0000	1.7
+++ linux/drivers/ide/legacy/q40ide.c	11 May 2006 23:52:07 -0000	1.8
@@ -80,6 +80,7 @@
 {
 	int i;
 
+	memset(hw, 0, sizeof(hw_regs_t));
 	for (i = 0; i < IDE_NR_PORTS; i++) {
 		/* BIG FAT WARNING: 
 		   assumption: only DATA port is ever used in 16 bit mode */
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

